cms/summary_probe

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+'''
+Probe to check used space.
+'''
+
+import sys
+import traceback
+from rucio.core import monitor
```

Comment:
Fernando, please for this and all future probes please make them use prometheus and prometheus only is fine. We are trying to get rid of the statsd stuff. 

I'm not sure if this comment applies to "common" probes, we'd have to check with @dchristidis but for sure CMS has no interest anymore in statsd. 

In this case then, RSE, account, and type of space should all be labels.

Reply:
Prometheus implemented.